assets/i18n/en.toml

```diff
@@ -45,28 +45,28 @@ savingsTotalEnergy = "{total} kWh charged"
 tabTitle = "Live community"
 power = "Charging power"
 powerSub1 = "{activeClients} of {totalClients} participants"
-powerSub2 = "are charging right now"
+powerSub2 = "charging…"
 greenShare = "Solar share"
-greenShareSub1 = "power is covered by"
-greenShareSub2 = "PV & battery storage"
-greenEnergy = "Solar energy"
+greenShareSub1 = "power derived from"
+greenShareSub2 = "solar, and battery storage"
+greenEnergy = "Solar"
 greenEnergySub1 = "charged with evcc"
 greenEnergySub2 = "since October 2022"
 
 [footer.telemetry]
-optIn = "I want to contribute my data as well."
-optInMoreDetails = "More details are available {0}."
+optIn = "I want to contribute my data."
+optInMoreDetails = "More details at {0}."
 optInMoreDetailsLink = "here"
 optInSponsorship = "Sponsoring required."
 
 [footer.sponsor]
 titleSponsor = "You are a supporter"
 titleNoSponsor = "Support us"
-thanks = "Thanks for your support, {sponsor}! It helps us with the further development."
+thanks = "Thank you, {sponsor}! You have helped the development."
 confetti = "Ready for confetti?"
-supportUs = "Our mission: Make solar charging the standard. Help us and support evcc financially."
-sticker = "...or evcc stickers?"
-confettiPromise = "There will be stickers and digital confetti ;)"
+supportUs = "Our mission is to make solar the norm. Help evcc by paying what it is worth to you."
+sticker = "… or evcc stickers?"
+confettiPromise = "You get stickers and digital confetti \;)"
```

Comment:
Tried to escape this since the smiley isn't making it into https://hosted.weblate.org/translate/evcc/evcc/en/?checksum=844a7935743b5aee

Reply:
Had to remove the escape because our parser struggled with it. In the UI it works as an old school ASCII smily without escaping.